pkg/api/install/install.go

```diff
@@ -56,6 +54,15 @@ func init() {
 		return
 	}
 
+	registered.RegisterVersions(availableVersions...)
```

Comment:
This should be done first.  You always want to register even if you have no external versions enabled.  In this order, you might return before registering.